app/sanitizers/feed_markdown_scrubber.rb

```diff
@@ -0,0 +1,13 @@
+class FeedMarkdownScrubber < Rails::Html::PermitScrubber
+  def initialize
+    super
+    self.tags = MarkdownProcessor::AllowedTags::FEED
+    self.attributes = MarkdownProcessor::AllowedAttributes::FEED
+  end
+
+  def allowed_node?(node)
+    return true if @tags.include?(node.name) && node.name != "a"
```

Comment:
I see you're using an `attr_accessor :tags` but then referencing `@tags` could you help me understand why not `tags`?

Reply:
I had copied this over from `rendered_markdown_scrubber.rb` as a starting point, and that was the way that Andy had done it originally.  Switching it to `tags` works too, so I don't have a problem with this change.